qiita_pet/handlers/study_handlers/base.py

```diff
@@ -78,9 +78,6 @@ def get(self):
         atype = self.get_argument('artifact_type')
         pt_id = self.get_argument('prep_template_id')
 
-        res = study_files_get_req(study_id, pt_id, atype)
+        res = study_files_get_req(self.current_user.id, study_id, pt_id, atype)
```

Comment:
For consistency with the other calls - when these calls where put in place, it was assumed that they'll be replaced by REST api calls, hence the idea on just passing the ids. I don't think that is gonna happen, b/c this calls are specialized for the interface but I want to be consistent at this point.